docs/release-notes/8.5.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[release-notes-header-8.5.0]]
+== 8.5
+
+[discrete]
+[[release-notes-8.5.0]]
+=== 8.5.0
+
+[discrete]
+[[known-issue-8.5.0]]
+==== Known issues
+* Users might experience slightly longer installation and upgrade times for the user and host risk score features ({pull}142434[#142434]).
+
+[discrete]
+[[breaking-changes-8.5.0]]
+==== Breaking changes
+// tag::breaking-changes[]
+// NOTE: The breaking-changes tagged regions are reused in the Elastic Installation and Upgrade Guide. The pull attribute is defined within this snippet so it properly resolves in the output.
+:pull: {pull}
+* Host and user risk score features that were installed in 8.4 or earlier are not ECS-compatible and therefore cannot generate new risk scores in 8.5.0. Before upgrading, users can archive their existing risk indices if they want to keep their "old" host and user risk scores. Otherwise, new risk indices will be generated once users upgrade host and user risk score features.
```

Comment:
PR here: https://github.com/elastic/kibana/pull/140377

Reply:
Thanks for the review, @angorayc ! To clarify, are you suggesting that we add the PR to the summary like so:

```suggestion
* Host and user risk score features that were installed in 8.4 or earlier are not ECS-compatible and therefore cannot generate new risk scores in 8.5.0. Before upgrading, users can archive their existing risk indices if they want to keep their "old" host and user risk scores. Otherwise, new risk indices will be generated once users upgrade host and user risk score features ({pull}140377[#140377]).
```

Reply:
Yes, this pr deals with the upgrade and installation process, should be helpful for as a reference here. Thank you.

Reply:
Thanks! I'll add this now.